vispy/visuals/tests/test_markers.py

```diff
@@ -13,10 +13,10 @@ def test_markers():
     # work for people
     with TestingCanvas() as c:
         marker = Markers()
-        marker.set_data(np.array([[50, 50]], np.float32))
+        np.random.seed(57983)
+        marker.set_data(np.random.normal(size=(30, 2), loc=50, scale=10))
```

Comment:
FYI you can use `rng = np.random.RandomState(57983)` then `rng.normal(...)` to avoid needing to use globals at all